packages/sit-onyx/src/components/OnyxNavBar/OnyxNavBar.vue

```diff
@@ -71,15 +71,20 @@ const {
 const isBurgerOpen = ref(false);
 const isContextOpen = ref(false);
 
-const isMobile = computed(() => {
-  const mobileWidth =
-    typeof props.mobileBreakpoint === "number"
-      ? props.mobileBreakpoint
-      : ONYX_BREAKPOINTS[props.mobileBreakpoint];
-  return width.value !== 0 && width.value < mobileWidth;
+const isMobileWidth = (mobileWidth: number) => width.value !== 0 && width.value < mobileWidth;
```

Comment:
The reactive `width` is used here so this should be a computed, otherwise it will not be re-computed when the width changes :)